flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
why no aarch64?

Reply:
we dont build or test aarch64.  im not sure it would work

Reply:
shadps4 doesnt recompile the games themselves.  theyre compiled as x86_64 originally and there is no emulation currently that translates that.